### ISSUE TITLE
OA: failed to parse

### ISSUE THREAD

**Issue by jvendetti** [ncbo/bioportal-project#34]
User [reported on the support list](http://ncbo-support.2288202.n4.nabble.com/bioontology-support-BioPortal-Feedback-from-michelle-td4655708.html) that they created the [OA ontology](http://bioportal.bioontology.org/ontologies/OA) in BioPortal, and none of the submissions have parsed.

**Comment by jvendetti** [comment]
There are two problems here:

**Bad pull URL**

I noticed is a bad pull URL that points to an [HTML page in GitHub](https://github.com/w3c/web-annotation/blob/gh-pages/vocab/wd/ontology/oa.rdf). The end user needs to enter a pull URL that points to the actual ontology sources. In GitHub, this would be accomplished by clicking the "Raw" button, which is a link to the raw ontology sources. Notice the difference in the URLs (the 2nd one being the correct one):

https://github.com/w3c/web-annotation/blob/gh-pages/vocab/wd/ontology/oa.rdf
https://raw.githubusercontent.com/w3c/web-annotation/gh-pages/vocab/wd/ontology/oa.rdf

**Invalid format**

This ontology is only available in RDF format, which our system won't handle. I verified this by creating a new submission with the correct pull URL and indeed it fails to parse.

The ontology source file contains a property "prov:wasRevisionOf" with a value of "http://www.openannotation.org/spec/core/20130208/oa.owl". I checked this URL and it's still valid, which means that an OWL version did exist at one time. Will contact the author to see if they are interested in trying to upload the older version.
